components/Category/Category.js

```diff
@@ -0,0 +1,369 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { fridgeList, shelfList, freezerList } from "../../itemlist";
```

Comment:
I'd suggest calling these `initialSomething` to make clear that this is only the initial state value.

```suggestion
import { initialFridgeList, initialShelfList, initialFreezerList } from "../../itemlist";
```